tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp

```diff
@@ -1768,6 +1813,12 @@ static void printCursorInfo(sourcekitd_variant_t Info, StringRef FilenameIn,
   for (auto Receiver : Receivers)
     OS << Receiver << '\n';
   OS << "RECEIVERS END\n";
+  OS << "SECONDARIES BEGIN\n";
+  for (auto Secondary : Secondaries) {
+    Secondary.print(OS, " | ", Filename, VFSFiles);
```

Comment:
This seemed like the easiest print for FileCheck. Happy for other opinions.

I was also thinking that we could change this print so that:
1. We always output everything, rather than only when it's set
2. Prefix with some key

Eg.
kind: the.symbol.kind
usr: someusr
isSystem: true/false

Then we could use CHECK-DAG rather than CHECK-NEXT (since CHECK-NOT wouldn't work how we'd want with  DAG). Turned out I only had to fix a few tests with the re-ordering though, so I left that for now.